src/CoreBundle/private/library/classes/TCMSMail.class.php

```diff
@@ -459,13 +459,15 @@ public function setMailPeerSecurity(HttpsContextInterface $mailPeerSecurity)
      */
     public function setSmtpHost($smtpHost)
     {
-        if (false === $colonPos = strpos($smtpHost, ':')) {
-            $this->smtpHost = $smtpHost;
-            $this->smtpPort = 25;
-        } else {
-            $this->smtpHost = substr($smtpHost, 0, $colonPos);
-            $this->smtpPort = (int) substr($smtpHost, $colonPos + 1);
-        }
+        $this->smtpHost = $smtpHost;
+    }
+
+    /**
+     * @param int $port
+     */
+    public function setSmtpPort($port)
```

Comment:
Please add a type declaration instead of the docblock, and avoid return type declaration.

Reply:
I am keeping with the style of the class here - change it anyway?

Reply:
I'd say we add typing anyway - if we keep the style, it is really hard to improve (we already deliberately broke the legacy conventions of capital letters for method names and the Hungarian notation).

Reply:
ok